### PR TITLE
GraphQL CV

### PR DESCRIPTION
**Pull request by jberglinds**
Update cv actions to use new graphql api

### CODE REVIEW

File: frontend/app/api.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const baseUrl = 'http://localhost:5040'
+const graphqlUrl = '/graphql'
```
Comment (aronstrandberg) [inline frontend/app/api.js]: Out of scope for this PR, but I think we should use `SCREAMING_SNAKE_CASE` for constants. opinions? @jeppes @Tiseno

File: frontend/app/containers/CvEdit/actions.js [hunk 2]
```diff
@@ -72,6 +73,7 @@ export function getRequest() {
 }
 
 export function getSuccess(cv) {
+  console.log(cv)
```
Comment (aronstrandberg) [inline frontend/app/containers/CvEdit/actions.js]: Remove console.log

File: frontend/app/api.js [hunk 3]
```diff
@@ -110,19 +112,42 @@ export function fetchUsers() {
...
+  const query = `{
+    cv { ${CV_FIELDS} }
+  }
+  `
+  const url = `${baseUrl}${graphqlUrl}?query=${query}`
+  return ftch(url, { ...credentials() })
+    .then(res => Promise.resolve(res.data.cv))
 }
 
 export function updateCv(id, cv) {
```
Comment (aronstrandberg) [inline frontend/app/api.js]: Also out of scope, but in the future I think we should split this file into several. Like one file per feature (cv, login, events etc) perhaps

File: frontend/app/api.js (hunk 1 above)
Comment (jeppes) [reply frontend/app/api.js]: hahah absolutely

File: frontend/app/api.js (hunk 3 above)
Comment (jeppes) [reply frontend/app/api.js]: 👍 yes, @jberglinds and I discussed this yesterday. Will be done at some point

File: frontend/app/containers/CvEdit/actions.js (hunk 2 above)
Comment (jeppes) [reply frontend/app/containers/CvEdit/actions.js]: 👍

**Pull request merged**